src/services/database.js

```diff
@@ -0,0 +1,45 @@
+const mongoose = require("mongoose");
+
+const Blogposts = require("../models/blogposts");
+const { dbUrl } = require("../../config/secrets");
+
+mongoose.connect(
+  dbUrl,
+  { useNewUrlParser: true }
+);
+
+function checkForWhitespace(post) {
+  let counter = 320;
+
+  while (post.charAt(counter) !== " " && counter <= post.length) {
+    counter += 1;
+  }
+  return counter;
+}
+
+function addSummary(blogpost) {
+  blogpost.forEach(element => {
+    element.summary = element.content.substring(
+      0,
+      checkForWhitespace(element.content)
+    );
+    element.summary += " ...";
+  });
+
+  return blogpost;
+}
+
+const allPosts = Blogposts.find({})
+  .then(blogposts => {
+    const postWithSummary = addSummary(blogposts);
+    const addObject = { blogs: postWithSummary };
+    return addObject;
+  })
+  .catch(err => console.log(err));
+
+const singlePosts = paramsId =>
```

Comment:
For consistency, I'd also change checkForWhitespace and addSummary to use the new function syntax. Or change allPosts and singlePosts.
Btw, it's a singlePost, isn't it?
And instead of paramsId, which is an artifact, you could use just id :)